matRad_generateStf.m

```diff
@@ -112,8 +114,12 @@
        error('at least one available peak position is negative - inconsistent machine file') 
     end
 
-    clear machine;
-
+    %clear machine;
+    
+    % define default foki indices
+    LowerFokiIdx  = 1; %[index]
+    UpperFokiIdx  = 2; %[index]
+    ThresholdFoki = 6; %[mm]
```

Comment:
If we implement this with two focus indices then we can never jump from 1 to 2 and then to 3 if also the sencond focus size gets to small. Lets wait for Benjamins response and see how we need to implement this. Generally it will not require a lot of time if we just search for the first focus index that exceeds ThresholdFoki.

BTW Lets use English spelling for foCus with a 'c' ;)